internal/api.go

```diff
@@ -110,6 +110,9 @@ func (a *defaultAPI) RemoveMetric(metric api.TaggedMetric) error {
 }
 
 func (a *defaultAPI) ToGraphiteName(metric api.TaggedMetric) (api.GraphiteMetric, error) {
+	if metric.MetricKey == "#graphite" {
```

Comment:
this should be a constant and documented somewhere. also, let's use `$` as special tags. (i've done it few times in the front-end for the UI styling issues.